src/Microsoft.VisualStudio.Threading.Analyzers/build/AdditionalFiles/vs-threading.LegacyThreadSwitchingMembers.txt

```diff
@@ -0,0 +1,8 @@
+﻿[Microsoft.VisualStudio.Shell.ThreadHelper]::Invoke
+[Microsoft.VisualStudio.Shell.ThreadHelper]::InvokeAsync
+[Microsoft.VisualStudio.Shell.ThreadHelper]::BeginInvoke
```

Comment:
Can you add/move these three lines to the vssdk-analyzers project? All VS-specific data is migrating there so this library is more VS-agnostic.

Reply:
I would prefer to leave this as a follow-up item since the behavior in this pull request is simply a preservation of behavior that already existed. Is that acceptable?

Reply:
Ok

Reply:
@sharwell, now that https://github.com/Microsoft/VSSDK-Analyzers/pull/73 has merged, can you prepare another PR for this repo that removes these from the master branch?

Reply:
Submitted #372